### ISSUE TITLE
CNTK_101_LogisticRegression.ipynb Getting TypeError exception on Trainer() call

### ISSUE THREAD

**Issue by IanMichaelAsh** [Microsoft/CNTK#1590]
Hi,

I'm trying to get the logistic regression tutorial to run, but I am getting a "TypeError was unhandled by user code" error on the call to Trainer() function, i.e on the line:

`trainer = Trainer(z, (loss, eval_error), [learner])`

The exception includes the message "__init__() missing 1 required positional argument: 'parameter_learners'"

Does anyone know why this exception is occurring and how I can fix it?

Thanks in advance, and apologies for what I'm sure is a very basic question,
Ian

**Comment by IanMichaelAsh** [comment]
After looking at other tutorials, the above line in the Tutorial seems to be badly formed and should be:

`trainer = Trainer(z, loss, eval_error, [learner])`
